### PR TITLE
Footer form and styling

### PR DESCRIPTION
**Pull request by fasaldana**
- Added a Form section at the end of the index.html file for the contact information.
- Added form.css for styling the contact form.
- Fixed buttons colors and hover actions.

### CODE REVIEW

File: footer.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
+footer {
```
Comment (nicupop729) [inline footer.css:1]: We should stick to the figma design as much as possible.

Here is how your design looks like:
image

And here is how the figma design looks like:
image

- [ ] Please take measures to improve your design to look more like the figma design by adding more space between sections, by changing the text color in the buttons, by removing buttons border, etc.

File: footer.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  display: grid;
```
Comment (nicupop729) [inline footer.css:2]: "In order to layout the elements on the page you should use Flexbox."

- [ ] As the project requirement specifically ask for using Flexbox, kindly refactor the display of the form.

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Required Changes ♻️
- [ ] Add `node_modules/` to `.gitignore` file;
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @fasaldana 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**